egs/librispeech/ASR/pruned_transducer_stateless5/conformer.py

```diff
@@ -1072,7 +1077,6 @@ class RandomCombine(nn.Module):
     def __init__(
         self,
         num_inputs: int,
-        num_channels: int,
```

Comment:
Please also remove the following doc:
https://github.com/k2-fsa/icefall/blob/8761452a2c6f5fba0617f0172ede4019e7b83b13/egs/librispeech/ASR/pruned_transducer_stateless5/conformer.py#L1067-L1069